pkgs/by-name/1f/1fps/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  xorg,
+}:
+buildGoModule rec {
+  pname = "1fps";
+  version = "0.1.10";
+
+  src = fetchFromGitHub {
+    owner = "1fpsvideo";
+    repo = "1fps";
+    rev = "v${version}";
+    hash = "sha256-3uPGFxEWmKQxQWPmotZI29GykUGQDjtDjFPps4QMs0M=";
+  };
+
+  proxyVendor = true;
+
+  vendorHash = "sha256-J3RGQhjpGURmXOwq19BbbNg5ERrUXHnSG5Id6gX7Nug=";
+
+  buildInputs = [
+    xorg.libX11
+    xorg.libXtst
+    xorg.libXi
+  ];
+
+  meta = {
+    description = "Encrypted Screen Sharing";
+    homepage = "https://1fps.video";
+    license = lib.licenses.fsl11Asl20;
+    maintainers = with lib.maintainers; [ renesat ];
```

Comment:
mssing `mainProgram`

Reply:
That's also in the direction of keeping the original name. That way you don't have to add this field.

Reply:
Don't you get a warning if it's unset?

Reply:
Oh, yeah. I just didn't try to use `nix run .#1fps`. Only build and run the binary